jupyter-widgets-schema/jupyterwidgetmodels.md

```diff
@@ -661,6 +661,23 @@ Attribute        | Type             | Default          | Help
 `placeholder`    | string           | `'\u200b'`       | Placeholder text to display when nothing has been typed
 `value`          | string           | `''`             | String value
 
+### Jupyter.Password
```

Comment:
Can you delete the changes to this file? This file is documenting the widgets as of 2.1.0 and is automatically generated. We'll generate another listing when we release the next version.